### ISSUE TITLE
Add some of the capabilities from Brackets

### ISSUE THREAD

**Issue by lszomoru** [microsoft/vscode-livepreview#169]
Extracting from https://github.com/microsoft/vscode-livepreview/issues/165

> ... some features from Brackets (which is no longer supported by Adobe) like selector highligting, live CSS updating, clicking on element in the browser moves cursor to the position in the source, etc... Or you can add these features into this extension :)

**Comment by timharris777** [comment]
I agree with this. Just tried live preview to do some html templating and thought, "man, I'm going to download brackets real quick to do this work because the live preview highlighted selected elements for easy visualisation." I went to download brackets and apparently it's no longer under development. I would love for this to be implemented.

**Comment by andreamah** [comment]
Live highlighting is tracked in #149

**Comment by andreamah** [comment]
Follow-along scroll tracked in #214
